source/authentication/nsf-access.rst

```diff
@@ -0,0 +1,40 @@
+.. _nsf-access:
+
+NSF ACCESS
+----------
+
+If your site is a part of the `National Science Foundation`_'s (NSF)
+`ACCESS`_ program (formerley `XSEDE`_) you can use their Identity Provider (IDP)
+to authenticate users for your Open OnDemand instance.
+
+OIDC Client Registration
+************************
+
+You should read the `ACCESS IDP documentation`_ on how to register your Open OnDemand
+instance as an Open ID Connect (OIDC) client.
+
+Once you've registered your Open OnDemand instance, you can then configure it accordingly.
+Since `ACCESS`_ uses Open ID Connect (OIDC) you can see our :ref:`oidc documentation <authentication-oidc>`
+for more details on how to configure Open OnDemand with what CI Logon has provided in
+registering your application.
+
+Mapping Users
+*************
+
+`ACCESS`_ users have allocations on many `ACCESS`_ resource, of which you are one.
+This means they have disperate usernames on all these systems and a unique username
```

Comment:
disperate is spelled wrong.  It should be disparate.

Reply:
@johrstrom .

Once the misspell is corrected, this looks good to me.

Reply:
thanks!